docs/serving/samples/cloudevents/cloudevents-nodejs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "knative-serving-cloudevents",
+  "version": "1.0.0",
+  "description": "Simple CloudEvents sample in Node.js",
+  "repository": "https://github.com/knative/docs",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "production": "npm start",
+    "dev": "nodemon index.js",
+    "lint": "standard --fix"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "cloudevents-sdk": "cloudevents/sdk-javascript#fd99cb1",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.3"
+  },
+  "devDependencies": {
+    "standard": "^14.3.4"
+  }
+}
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
}

```

Reply:
my vscode editor keeps trying to be helpful and removing new line